azurerm/resource_arm_cosmos_db_account.go

```diff
@@ -77,6 +77,26 @@ func resourceArmCosmosDBAccount() *schema.Resource {
 				),
 			},
 
+			"is_virtual_network_filter_enabled": {
+				Type:     schema.TypeBool,
+				Optional: true,
+				Default:  false,
+			},
+
+			"virtual_network_rules": {
+				Type:     schema.TypeSet,
+				Optional: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"id": {
+							Type:     schema.TypeString,
+							Optional: true,
+						},
```

Comment:
Could we validate the ID here?

Reply:
Ok, added ValidateFunc: azure.ValidateResourceID